simphony_metadata/scripts/tests/test_validation.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+
+import numpy
+from mock import patch
+
+from .cuba import CUBA
+from .keywords import KEYWORDS
+from .meta_class import api
```

Comment:
@kitchoi I get the same error even on an ubuntu machine. I see you have patched this line on the other test, if I do the same it works fine. I'll push my change.